### ISSUE TITLE
IANAZone.isValidZone results depends on where it is run.

### ISSUE THREAD

**Issue by ShaneMcNamara** [moment/luxon#1393]
**Describe the bug**
Running IANAZone.isValidZone('US/Central') on the browser (Chrome) returns true, while running it on a Node.js server returns false.

**Actual vs Expected behavior**
I would expect the results to be consistent.

**Desktop (please complete the following information):**

- OS: MacOS 13.1
- Browser: Chrome Version 110.0.5481.177
- Luxon version: 3.3.0
- Your timezone: America/Vancouver

**Additional context**
We have functionality where the user saves their time zone. Later, on the backend we send a PDF report to them based off certain events and we need to write a human readable time (in their timezone) inside the PDF. Our servers are not necessarily in the same timezone as the user, so we pull the timezone from our DB and try to use `IANAZone.create(tz)`  but this will fail if they selected certain timezones (e.g. US/Central)

**Comment by icambron** [comment]
"US/Central" works fine for me in Node v19.7.0. I also tried it on v17.1.0, and that worked too. So a few things to note:

 1. The JS environment provides the ICU data that contains the time zone information. Perhaps you have an old version of Node, or perhaps your copy of Node wasn't build with ICU data. Either way, Luxon can't control this.
 2. "US/Central" is a strange way to express a zone. "US/Central" is an alias for "America/Chicago", which is the more usual string to use.

**Comment by ShaneMcNamara** [comment]
Thanks @icambron, I must have missed the [ICU note for install](https://github.com/moment/luxon/blob/master/docs/install.md). It does work correctly when I upgrade Node.

Re #2. I get my timezone names from moment-timezone (`moment.tz.names()`), as my understanding is that Luxon does not provide a list of timezone names. Is that understanding correct?

**Comment by diesieben07** [comment]
Correct, Luxon does not provide a list of time zones, as it doesn't ship with its own time zone data. Luxon relies on the Intl API (mainly Intl.DateTimeFormat) and it does not expose a list  of time zones.

**Comment by dasa** [comment]
You can get a list using `Intl.supportedValuesOf("timeZone")`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf

**Comment by diesieben07** [comment]
Good tip, @dasa, I learned something new today. But do keep in mind that this API is fairly new and only supported in the most recent browser versions.
